examples/scaling-criteo/02-ETL-with-NVTabular.ipynb

```diff
@@ -551,14 +491,15 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "CPU times: user 3.36 s, sys: 805 ms, total: 4.17 s\n",
-      "Wall time: 1min 58s\n"
+      "CPU times: user 7.59 s, sys: 2.1 s, total: 9.7 s\n",
+      "Wall time: 9min 28s\n"
      ]
     }
    ],
    "source": [
     "%%time\n",
     "workflow.transform(train_dataset).to_parquet(\n",
+    "    output_files=5,\n",
```

Comment:
We might want to explicitly set the number of output files to be the same as the number of workers.  Since the `output_files` approach is forcing a depth-first execution of each output-file subgraph, an 8-GPU system will not be fully utilized if this is set to `5`.